src/devices/cpu/z80/z80.cpp

```diff
@@ -3356,7 +3287,7 @@ void z80_device::nomreq_ir(s8 cycles)
 
 void z80_device::nomreq_addr(u16 addr, s8 cycles)
 {
-	for (; cycles; cycles--)
+	while(cycles--)
 	{
```

Comment:
`while` isn’t a function – it should have a space like the other control flow keywords like `while (...)`